### ISSUE TITLE
[VITA] [Bug?] RGUI interface uses bilinear filtering by default without an option to disable it

### ISSUE THREAD

**Issue by AltPlus30** [libretro/RetroArch#15967]
## Description

When switching to RGUI the whole UI is blurry. Probably because it doesn't render at native resolution and fails to scale properly. Options to disable Bilinear Filtering or use Integer Scaling don't do anything. This isn't an issue on other platforms.

### Expected behavior

Sharp pixels in menus or an option to toggle it.

### Actual behavior

The text is too big and blurry.

### Steps to reproduce the bug

1. Install RetroArch on PS Vita
2. Switch to RGUI using either Drivers or User Interface menu

### Bisect Results

Basically, it's been happening since RGUI introduction on Vita.

### Version/Commit

- RetroArch: 1.16.0 Stable

### Environment information

- OS: PS Vita

**Comment by zoltanvb** [comment]
Is this the same as what you are seeing?
![2025-01-12-202027](https://github.com/user-attachments/assets/b90498f6-cb05-46d4-9a14-732e87a77dfb)
"Too big" text is expected, the screen size is fixed for rgui I believe.

**Comment by zoltanvb** [comment]
Hmm.
`[INFO] [Video]: Set video size to: 960x720.`
That seems a bit off, considering the native resolution of 960x544.

**Comment by AltPlus30** [comment]
> Is this the same as what you are seeing?
Yeah, pretty much.

> "Too big" text is expected, the screen size is fixed for rgui I believe.
It's not only that the text is big but it's also blurry because of the filtering (or improper scaling). Meanwhile, the upscaling option only works on piglet build.

**Comment by zoltanvb** [comment]
Not quite sure what changed, but current build with a fresh configuration will display more lines for rgui and does not look blurry any more to me.

**Comment by zoltanvb** [comment]
Ehh, blurriness is still there, just less noticeable. But comparing it to an Android device, it is visible.
